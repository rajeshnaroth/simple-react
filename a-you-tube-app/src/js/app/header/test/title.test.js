import React from "react"
import renderer from "react-test-renderer"

import Title from "../title"

test("Title gets created", () => {
  const component = renderer.create(<Title message="Hello World" />)
  expect(component.toJSON()).toMatchSnapshot()
})
