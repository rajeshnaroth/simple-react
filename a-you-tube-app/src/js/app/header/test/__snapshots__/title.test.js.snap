// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title gets created 1`] = `
<div
  className="sc-bdVaJa cWPLMR"
>
  <h1 />
</div>
`;
